src/components/button/JustifyCenterButton.tsx

```diff
@@ -0,0 +1,26 @@
+// Chakra imports
+import { useColorModeValue, Button, Tooltip } from "@chakra-ui/react";
+import { MdFormatAlignJustify } from "react-icons/md";
+// Custom components
+
+export default function Default() {
+	const textColor = useColorModeValue("secondaryGray.900", "white");
+	const boxBg = useColorModeValue("secondaryGray.300", "whiteAlpha.100");
+
+	const handleJustifyCenterClick = () => {
+		document.execCommand("justifyCenter", true, "");
+	};
+
+	return (
+		<Tooltip
+			label="Justify Center"
```

Comment:
I think this would be "Align Center" not justify center, justify is another independent alignment. 

This fix need to be placed in file names, function names and so on.

Reply:
fixed in https://github.com/1-Harshit/ed-eater/pull/20/commits/220f9f1f200d9cd4afc28f86575c011f3f56dadf